fuzz/Cargo.toml

```diff
@@ -16,12 +16,13 @@ afl_fuzz = ["afl"]
 honggfuzz_fuzz = ["honggfuzz"]
 
 [dependencies]
+afl = { version = "0.3", optional = true }
 lightning = { path = "..", features = ["fuzztarget"] }
 bitcoin = { version = "0.13", features = ["fuzztarget"] }
-secp256k1 = { version = "0.9", features=["fuzztarget"] }
-rust-crypto = "0.2"
+hex = "0.3.2"
```

Comment:
I think usually its best to just specify 0.3, as minor bumps we'd want to take as bugfixes, no?

Reply:
fair point!